test/ergw_test_lib.erl

```diff
@@ -23,7 +23,8 @@
 	 gtp_context_inc_restart_counter/1,
 	 gtp_context_new_teids/1, gtp_context_new_teids/3,
 	 make_error_indication_report/1]).
--export([start_gtpc_server/1, stop_gtpc_server/1, stop_gtpc_server/0,
+-export([start_gtpc_server/1, start_gtpc_server/2,
+	 stop_gtpc_server/1, stop_gtpc_server/0, stop_gtpc_server_node/1,
```

Comment:
For symmetry reason I would suggest the name
stop_gtpc_server/1,

Reply:
Done